integration-tests/src/test/java/com/ca/mfaas/discoverableclient/DownloadApiIntegrationTest.java

```diff
@@ -36,6 +36,7 @@ public void shouldDoGetRequestAndDownloadZipFile() {
             expect().
             statusCode(200).
             header("Content-Disposition", "attachment;filename=sample-text.zip").
+            header("Content-Encoding", "gzip").
             when().
             get(uri)
             .then().contentType("application/zip")
```

Comment:
i believe we can also add Content-Encoding - "gzip" check as well.
.then().contentType("application/zip")
.contentEncoding("gzip") as well

![image](https://user-images.githubusercontent.com/4791635/71832610-50067d00-3079-11ea-814f-f315a86737fd.png)

Reply:
The header is on the expect() block, so I believe they serve the same purpose as then(). It is just different syntax. 
As @JirkaAichler asked in previous question, when setContentCompressionEnabled(false) is removed the integration test fails.

Reply:
hi, just making sure we are checking in response received is gzip, as depicted in screenshot i posted above. As far as that's covered, we are good. 
And failing when setContentCompressionEnabled(false) is removed, sound good to me as well. 
Thanks again!